src/main/scala/nl.knaw.dans.easy.deposit/servlets/AuthServlet.scala

```diff
@@ -16,11 +16,20 @@
 package nl.knaw.dans.easy.deposit.servlets
 
 import nl.knaw.dans.easy.deposit.EasyDepositApiApp
+import nl.knaw.dans.easy.deposit.authentication.AuthUser
 import nl.knaw.dans.easy.deposit.logging._
+import nl.knaw.dans.lib.error._
 import org.scalatra.NoContent
 
 class AuthServlet(app: EasyDepositApiApp) extends AbstractAuthServlet(app) {
 
+  override protected def fromSession: PartialFunction[String, AuthUser] = {
```

Comment:
Would like to keep this method somehow in the authentication package, like registerAuthStrategies in ProtectedServlet below.